R/zzz.R

```diff
@@ -0,0 +1,3 @@
+.onUnload <- function (libpath) {
+  library.dynam.unload("frogger", libpath)
```

Comment:
I never use this - what added it? Looks like some packages use it and others don't - I wonder what effect it has...

Reply:
I got this from https://r-pkgs.org/src.html#c-best-practices